tests/upgrade/upgrade_test.go

```diff
@@ -65,7 +65,7 @@ var _ = Describe("{UpgradeVolumeDriver}", func() {
 
 		Step("reinstall and validate all apps after upgrade", func() {
 			for i := 0; i < Inst().GlobalScaleFactor; i++ {
-				contexts = append(contexts, ScheduleApplications(fmt.Sprintf("upgradevolumedriverreinstall-%d", i))...)
+				contexts = append(contexts, ScheduleApplications(fmt.Sprintf("upgradedvolumedriver-%d", i))...)
```

Comment:
I think it will not schedule new applications as namespaces will be the same as previous scheduled application

Reply:
the prefix is different than the initial one. upgradevolumedriver vs upgrade`d`volumedriver.
that would make the app new right?

Reply:
ah, missed that